apps/storefront/.lintstagedrc.json

```diff
@@ -1,4 +1,3 @@
 {
-  "*.{ts,tsx,js}": "eslint --fix",
-  "*.{js,jsx,ts,tsx,md}": "npx cspell lint"
```

Comment:
? why remove the spell check?

Reply:
Is there a way I can commit without this rule? this is a really small change, if I leave it on It starts flagging a bunch of cases in my local environment and the PR would get bigger and kind of lose its purpose

Reply:
good callout ignacio. I discussed w/ victor - and have a plan of keeping spell check as the goal is to keep improving our spelling issues through each PR.

So the PR having changes outside it's original scope is expected (although if it gets out of hand we can ignore specific files/words/ come up with a better strategy other than disabling it fully)..

Reply:
Just added back the setting :+1: